.github/workflows/propose_translation.yml

```diff
@@ -0,0 +1,58 @@
+name: Propose Translation
+on:
+  workflow_call:
+    inputs:
+      branch:
+        type: string
+        default: ${{ github.ref_name }}
+      language:
+        type: string
+        default: ""
+      locale_folder:
```

Comment:
This means that only skills using the latest `locale` directory can use this workflow, correct?

Reply:
locales can be set up in every package, not only skills. They have the usual structure
`{{ inputs.locale_dir }}/<langcode>/`

The structure below the langcode dir is arbitrary and gets walked and mirrored for the target langs

Reply:
if you are alluding to the old structure (dialog/intent in the basefolder), those are moved to `./locale/` in the translation process